src/openrct2/interface/Window.cpp

```diff
@@ -37,6 +37,7 @@
 #include <functional>
 #include <iterator>
 #include <list>
+window_close_modifier last_close_modifier = { WindowClass::Null, 0, CloseWindowModifier::CLOSE_WINDOW_MODIFIER_NONE };
```

Comment:
I would move this to be under `gCurrentWindowColours`. If left here, there should at least be a gap between the includes and this.